### PR TITLE
Trg staging updates

### PR DESCRIPTION
**Pull request by markkramerus**
This PR addresses feedback from the TRG. 
1.	TRG approves of non-TNM profiles, in principle.
2.	They want to add prostate cancer gleason grade.
        a.	Scott Campbell to confirm which SNOMED codes to use for .code, .method, .value
3.	Add Breslow staging for melanoma
4.	Move Neuroblastoma to pediatric IG
5.	Remove ALL FAB Phase (outdated)
6.	Codes in CancerStagingStageType and Method involving Children’s Onco Group or St. Judes and other pediatric cancers: move to pediatric cancer IG
Still in flux:
7.	Ann Arbor and Cotswold not used much – potentially limit to Lugano – will await further input from TRG
8.	TRG will advise on whether ISS is still in use for melanoma -- we may end up getting rid of it.

### CODE REVIEW

**Review by saulakravitz — APPROVED** [review]
Minor comment:  I thought we were dropping enneking staging system/method.

File: input/fsh/VS_Staging.fsh [hunk 1]
```diff
@@ -146,42 +134,23 @@ Description: "Staging system or method used for staging cancers. The terms in th
...
 * SCT#1149131009 "Rai staging system for chronic lymphocytic leukemia (tumor staging)"
 * SCT#1149163003 "Revised International Staging System for multiple myeloma (staging scale)"
-* SCT#254365003 "Siopel liver staging system (tumor staging)"
-* SCT#254370005 "St Jude's malignant histiocytosis tumor staging system (tumor staging)"
-* SCT#254371009 "St Jude's non-Hodgkin's lymphoma staging system (tumor staging)"
 * SCT#254366002 "Stannard staging system for retinoblastoma (tumor staging)"
 * SCT#254382001 "Trophoblastic malignancy staging system (tumor staging)"
-* SCT#254368001 "United Kingdom children's cancer study group central nervous system tumor staging system (tumor staging)"
 * SCT#254378003 "Walter Reed testicular tumor staging system (tumor staging)"
 * SCT_TBD#USCRS-34949 "Enneking surgical staging system (tumor staging)" // requested June 2023
```
Comment (saulakravitz) [inline input/fsh/VS_Staging.fsh:146]: I thought we were dropping this

**Review by saulakravitz — DISMISSED** [review]
Minor comment:  I thought we were dropping enneking staging system/method.

File: input/fsh/VS_Staging.fsh (hunk 1 above)
Comment (markkramerus) [reply input/fsh/VS_Staging.fsh:146]: For the record, this VS is like a superset of what we might do at some later date. Including or excluding any codes reallly has no effect on the use or functionality of mCODE.

Comment (markkramerus) [reply input/fsh/VS_Staging.fsh:146]: Keep the requests going. We might need all of them, including the pediatric and surgical staging systems.

**Pull request merged**